Triaging-process.md

```diff
@@ -0,0 +1,73 @@
+## General
+This wiki outlines the steps to triage new and existing issues. This ensures that all issues are being effectively tracked and prioritized.
+
+### Steps to triage incoming issues coming into Oppia as a whole (first-stage triage):
```

Comment:
Probably mention "first-stage triage" at the beginning. So the reader knows that there are 2 stages.

Reply:
Done